kale/embed/image_cnn.py

```diff
@@ -48,6 +48,45 @@ def output_size(self):
         return self._out_features
 
 
+class _ADDneck(nn.Module):
+    """Simple network for domain specific embedding
```

Comment:
This description does not match the class name `add neck` very well. Can you consider change the class name or update the description so we know why it is called `add neck`?

Reply:
`domain specific embedding` could be a jargon to many so some explanation helps.

Reply:
I do not know how to call it properly. It is used for [MFSAN](https://github.com/easezyc/deep-transfer-learning/tree/master/MUDA/MFSAN) only, where it is name `ADDneck` in the [original implementation](https://github.com/easezyc/deep-transfer-learning/blob/master/MUDA/MFSAN/MFSAN_2src/resnet.py#L94).

Reply:
Thanks to @XianyuanLiu. It is a variant of [Bottleneck](https://github.com/pytorch/vision/blob/main/torchvision/models/resnet.py#L86). I have renamed it.